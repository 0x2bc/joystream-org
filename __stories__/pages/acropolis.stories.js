import React from 'react';
import { storiesOf } from '@storybook/react';

import AcropolisPage from '../../src/pages/acropolis';

storiesOf('Pages|Acropolis', module).add('default', () => <AcropolisPage />);
