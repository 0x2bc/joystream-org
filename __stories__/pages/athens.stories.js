import React from 'react';
import { storiesOf } from '@storybook/react';

import AthensPage from '../../src/pages/athens';

storiesOf('Pages|Athens', module).add('default', () => <AthensPage />);
