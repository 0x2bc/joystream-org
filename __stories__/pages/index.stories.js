import React from 'react';
import { storiesOf } from '@storybook/react';

import IndexPage from '../../src/pages/index';

storiesOf('Pages|Home', module).add('default', () => <IndexPage />);
