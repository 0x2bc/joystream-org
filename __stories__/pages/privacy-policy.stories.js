import React from 'react';
import { storiesOf } from '@storybook/react';

import PrivacyPolicyPage from '../../src/pages/privacy-policy';

storiesOf('Pages|Privacy Policy', module).add('default', () => <PrivacyPolicyPage />);
