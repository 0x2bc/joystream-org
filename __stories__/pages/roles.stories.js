import React from 'react';
import { storiesOf } from '@storybook/react';

import RolesPage from '../../src/pages/roles';

storiesOf('Pages|Roles', module).add('default', () => <RolesPage />);
