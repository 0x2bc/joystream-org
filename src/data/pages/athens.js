import React from 'react';
import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as StorageImage } from '../../assets/svg/storage.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as payoutImage } from '../../assets/svg/participation-payout.svg';
import { ReactComponent as blockImage } from '../../assets/svg/block-platform-content-files.svg';
import { ReactComponent as bookImage } from '../../assets/svg/platform-content-files.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: false,
      type: 'migration',
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: false,
      type: 'migration',
    },
    {
      image: StorageImage,
      title: 'Storage Provider',
      to: '/roles#Storage-Provider',
      key: 'storageProviders',
      hasLabel: true,
      type: 'migration',
    },
  ],
};

const analytics = [
  { title: 'Participation Payout', image: payoutImage, value: '$1576' },
  {
    title: 'Active Validators',
    image: ValidatorsImage,
    key: 'validatorsCount',
  },
  { title: 'Platform Content Files', image: bookImage, key: 'mediaFiles' },
  { title: 'Block Height', image: blockImage, key: 'blockHeight' },
];

const goals = [
  {
    title: 'Enable media on the platform',
    text:
      /* eslint-disable */
      'As a "user governed media platform", allowing users to upload and consume and content was a major goal for us.',
    /* eslint-disable */
  },
  {
    title: 'Introduce platform memberships',
    text:
      /* eslint-disable */
      'Although we are building an open blockchain in the sense that users are free to send tokens as they like, the Joystream experience is about having the platform users own, operate and govern the platform. We are agnostic on exactly what parts of the platform will be reserved for members, some actions will require and extra level of screening and accountability.',
    /* eslint-disable */
  },
  {
    title: 'Build and release the storage node',
    text:
      /* eslint-disable */
      'Although a storage node was built and used to host and distribute the platform content, it had some bugs making it unable to sync between clients and required a hardcoded liaison.',
    /* eslint-disable */
    state: 'postponed',
  },
  {
    title: 'Upgrade the runtime through a Council vote',
    /* eslint-disable */
    text: [
      'The intention was for Jsgenesis to create a proposal for a ',
      <a href="https://blog.joystream.org/upgrades/">runtime upgrade</a>,
      ', and have the Council vote on it. If the Council reached quorum, the consensus rules of the system would automatically get upgraded in flight. Unfortunately, the ',
      <a href="https://blog.joystream.org/sparta-sacked/">
        Sparta network crashed
      </a>,
      ' before we reached this stage, and had to start Athens as a new chain with a new genesis block.',
    ],
    /* eslint-disable */
    state: 'postponed',
  },
];

export { roles, analytics, goals };
