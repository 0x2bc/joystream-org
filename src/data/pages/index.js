import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as StorageImage } from '../../assets/svg/storage.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as ScreenerImage } from '../../assets/svg/screener.svg';
import { ReactComponent as MembershipCuratorImage } from '../../assets/svg/membership-curator.svg';
import { ReactComponent as ContentCuratorImage } from '../../assets/svg/content-curator.svg';
import { ReactComponent as ContentCreatorImage } from '../../assets/svg/content-creator.svg';
import { ReactComponent as BandwidthProviderImage } from '../../assets/svg/bandwidth-provider.svg';
import { ReactComponent as DiscoveryProviderImage } from '../../assets/svg/discovery-provider.svg';
import { ReactComponent as LiveStreamingProviderImage } from '../../assets/svg/live-streaming-provider.svg';
import { ReactComponent as BuilderImage } from '../../assets/svg/builder.svg';
import { ReactComponent as CommunicationModeratorImage } from '../../assets/svg/communication-moderator.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: false,
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: false,
    },
    {
      image: StorageImage,
      title: 'Storage Provider',
      to: '/roles#Storage-Provider',
      key: 'storageProviders',
      hasLabel: true,
    },
  ],
  future: [
    { image: ScreenerImage, title: 'Membership Screener', to: '/roles' },
    {
      image: MembershipCuratorImage,
      title: 'Membership Curator',
      to: '/roles',
    },
    { image: ContentCuratorImage, title: 'Content Curator', to: '/roles' },
    { image: ContentCreatorImage, title: 'Content Creator', to: '/roles' },
    {
      image: BandwidthProviderImage,
      title: 'Bandwidth Provider',
      to: '/roles',
    },
    {
      image: DiscoveryProviderImage,
      title: 'Discovery Provider',
      to: '/roles',
    },
    {
      image: LiveStreamingProviderImage,
      title: 'Live Streaming Provider',
      to: '/roles',
    },
    { image: BuilderImage, title: 'Builder', to: '/roles' },
    {
      image: CommunicationModeratorImage,
      title: 'Communication Moderator',
      to: '/roles',
    },
  ],
};

export { roles };
