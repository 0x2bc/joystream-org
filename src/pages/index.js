import React from 'react';
import { pagePropTypes } from '../propTypes';

import getApiPath from '../utils/getApiPath';
import mapStatusDataToAnalytics from '../utils/mapStatusDataToAnalytics';
import mapStatusDataToRoles from '../utils/mapStatusDataToRoles';

import withApi from '../components/_enhancers/withApi';

import BaseLayout from '../components/_layouts/Base';
import Link from '../components/Link';
import Subheader from '../components/Subheader';
import TestnetItem from '../components/TestnetItem';
import Analytics from '../components/Analytics';
import Button from '../components/Button';
import TitleWrapper from '../components/TitleWrapper';
import RoleList from '../components/RoleList';
import ColumnsLayout from '../components/ColumnsLayout';
import LayoutWrapper from '../components/LayoutWrapper';
import Hero from '../components/Hero';
import SiteMetadata from '../components/SiteMetadata';

import AcropolisImage from '../assets/svg/acropolis-main.svg';
import heroImage from '../assets/svg/hero-builder.svg';
import RomeImage from '../assets/svg/rome-main.svg';
import constantinopleImage from '../assets/svg/constantinople-main.svg';
import alexandriaImage from '../assets/svg/alexandria-main.svg';
import { ReactComponent as TickImage } from '../assets/svg/tick.svg';
import { ReactComponent as ClockImage } from '../assets/svg/clock.svg';

import { roles } from '../data/pages';
import { launchDate as alexandriaNetworkLaunchDate } from '../data/pages/alexandria';

import './style.scss';

const activeTestnet = {
  name: 'Alexandria',
  incentivesLink: 'https://www.joystream.org/testnet',
};

const IndexPage = ({ content }) => (
  <BaseLayout>
    <SiteMetadata
      title="Joystream: The video platform DAO"
      description="Joystream is a video platform controlled, owned, and operated by its users."
    />
    <Hero
      image={heroImage}
      title={
        <>
          The video <br /> platform DAO
        </>
      }
      animationStartValue={0}
    >
      <p className="IndexPage__hero-paragraph">
        Joystream is a video platform controlled, owned, and operated by its users.
      </p>
      <div className="IndexPage__hero-group">
        <Button noWrap large secondary className="IndexPage__hero-button" href="/testnet/">
          Earn Tokens
        </Button>
        <Button
          noWrap
          large
          secondary
          reversed
          className="IndexPage__hero-button"
          href="https://testnet.joystream.org/"
        >
          Launch UI
        </Button>
      </div>
    </Hero>

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children={
          <>Alexandria is our seventh testnet, introducing a variety of important technical improvements.</>
        }
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    <Analytics title="Testnet Metrics" large content={mapStatusDataToAnalytics(content)}>
      <Button secondary href={activeTestnet.incentivesLink}>
        Participate and Earn Monero
      </Button>
    </Analytics>

    {/*

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children="Alexandria is our seventh testnet, focused on a variety of important technical improvements."
        date={alexandriaNetworkLaunchDate}
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    */}

    <LayoutWrapper dark>
      <TitleWrapper title="Become a user, run the platform">
        <Subheader
          title="Active roles on the current testnet"
          content="learn more, join a role and subscribe for more"
          icon={TickImage}
        />
        <ColumnsLayout>
          <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>
        <Subheader
          title="Roles coming in future testnets"
          content="choose a role, learn more and subscribe to get updated"
          icon={ClockImage}
        />
        <ColumnsLayout columnsCount={3}>
          <RoleList roles={roles.future} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>

        <Button secondary to="/roles" className="IndexPage__roles-button">
          Learn more about the roles
        </Button>
      </TitleWrapper>
    </LayoutWrapper>
  </BaseLayout>
);

IndexPage.propTypes = pagePropTypes;

export { IndexPage };
export default withApi(IndexPage, getApiPath('/'));
