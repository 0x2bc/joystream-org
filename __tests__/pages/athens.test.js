import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import { AthensPage } from '../../src/pages/athens';

import testContent from '../../__mocks__/data/testContent';

configure({ adapter: new Adapter() });

describe('AthensPage page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AthensPage content={testContent} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
