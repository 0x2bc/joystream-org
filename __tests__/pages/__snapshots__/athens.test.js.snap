// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AthensPage page renders correctly 1`] = `
<div>
  <nav
    className="Navbar"
  >
    <div
      className="Navbar__container"
    >
      <a
        className="Link"
        href="/"
      >
        <icon-mock
          className="Navbar__logo"
        />
      </a>
      <div
        className="Navbar__links"
      >
        <a
          className="Navbar__link Navbar__link--highlighted Link Link--highlighted"
          href="https://www.jsgenesis.com/#openings"
          label="We are hiring!"
        >
          We are hiring!
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/"
          label="Blog"
        >
          Blog
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/manifesto/"
          label="Manifesto"
        >
          Manifesto
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/whitepaper/blob/master/paper.pdf"
          label="Whitepaper"
        >
          Whitepaper
        </a>
        <a
          className="Navbar__link Link"
          href="/roles"
          label="Roles"
        >
          Roles
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/joystream"
          label="Repository"
        >
          Repository
        </a>
      </div>
      <div
        className="Navbar__trigger"
        onClick={[Function]}
        role="presentation"
      />
    </div>
  </nav>
  <section
    className="Hero Hero--indented"
  >
    <div
      className="Hero__container"
    >
      <div
        className="Hero__content"
      >
        <h1
          className="Hero__title"
        >
          Athens Network
        </h1>
        <div
          className="Hero__description"
        >
          <p
            className="AthensPage__hero-paragraph"
          >
            Explore available roles and pick the one that suits you the most. Influence platforms development earning Monero in the process.
          </p>
          <div
            className="Card Card--info"
          >
            <p
              className="Card__header"
            >
              <icon-mock
                className="Card__icon"
              />
              Network replaced
            </p>
            <div
              className="Card__content"
            >
              <div
                className="DateCounter DateCounter--light DateCounter--large"
              >
                <p
                  className="DateCounter__title"
                >
                  AFTER LAUNCHING 17 / 04 / 19
                  <br />
                  THE NETWORK WAS UPGRADED TO 
                  <a
                    className="Link"
                    href="/acropolis"
                  >
                    ACROPOLIS
                  </a>
                   
                  ON
                </p>
                <div
                  className="DateCounter__container"
                >
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      24
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      day
                    </span>
                  </div>
                  <span
                    className="DateCounter__value, DateCounter__separator"
                  >
                    /
                  </span>
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      06
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      month
                    </span>
                  </div>
                  <span
                    className="DateCounter__value, DateCounter__separator"
                  >
                    /
                  </span>
                  <div
                    className="DateCounter__segment"
                  >
                    <span
                      className="DateCounter__value"
                    >
                      19
                    </span>
                    <span
                      className="DateCounter__label"
                    >
                      year
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="Hero__image-wrapper"
      >
        <button
          className="Chip"
          onClick={[Function]}
        >
          <icon-mock
            className="Chip__icon"
          />
          <span
            className="Chip__text"
          >
            What is this?
          </span>
        </button>
        <div
          className="Plx  "
          disabled={false}
          style={
            Object {
              "visibility": "hidden",
            }
          }
        >
          <icon-mock
            className="Hero__image"
          />
        </div>
      </div>
    </div>
  </section>
  <div
    className="LayoutWrapper"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Network Statistics
        </h2>
        <p
          className="TitleWrapper__subtitle"
        >
          As they were at the end of the network.
        </p>
        <section
          className="Analytics"
        >
          <div
            className="Analytics__container"
          >
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Participation Payout
              </p>
              <p
                className="AnalyticsItem__value"
              >
                $1576
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Active Validators
              </p>
              <p
                className="AnalyticsItem__value"
              >
                20
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Platform Content Files
              </p>
              <p
                className="AnalyticsItem__value"
              >
                9
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Block Height
              </p>
              <p
                className="AnalyticsItem__value"
              >
                1110117
              </p>
            </div>
          </div>
        </section>
      </section>
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Critical Documents
        </h2>
        
        <div
          className="ColumnsLayout"
          data-columns={2}
        >
          <div
            className="Pane Pane--disabled"
          >
            <div
              className="Pane__wrapper"
            >
              <icon-mock
                className="Pane__img"
              />
            </div>
            <div
              className="Pane__content"
            >
              <p
                className="Pane__title"
              >
                Full Specifications
              </p>
              <p
                className="Pane__paragraph"
              >
                No specifications was published for Athens.
              </p>
            </div>
          </div>
          <a
            className="Pane Link"
            href="https://github.com/Joystream/joystream/tree/master/testnets/athens"
            target="_blank"
          >
            <div
              className="Pane__wrapper"
            >
              <icon-mock
                className="Pane__img"
              />
            </div>
            <div
              className="Pane__content"
            >
              <p
                className="Pane__title"
              >
                Release Plan
              </p>
              <p
                className="Pane__paragraph"
              >
                Read the release plan as it was made during the planning stage, and learn more about how the development evolved.
              </p>
            </div>
          </a>
        </div>
      </section>
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Testnet Goals
        </h2>
        <p
          className="TitleWrapper__subtitle"
        >
          The goals below is a simplified representation of the Key Results listed in our Release
           
          <a
            className="Link"
            href="https://github.com/Joystream/joystream/tree/master/testnets/athens#okrs"
          >
            OKR
          </a>
        </p>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--achieved"
              >
                Achieved
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Enable media on the platform
              </p>
              As a "user governed media platform", allowing users to upload and consume and content was a major goal for us.
            </div>
          </div>
        </div>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--achieved"
              >
                Achieved
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Introduce platform memberships
              </p>
              Although we are building an open blockchain in the sense that users are free to send tokens as they like, the Joystream experience is about having the platform users own, operate and govern the platform. We are agnostic on exactly what parts of the platform will be reserved for members, some actions will require and extra level of screening and accountability.
            </div>
          </div>
        </div>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--postponed"
              >
                Postponed
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Build and release the storage node
              </p>
              Although a storage node was built and used to host and distribute the platform content, it had some bugs making it unable to sync between clients and required a hardcoded liaison.
            </div>
          </div>
        </div>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--postponed"
              >
                Postponed
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Upgrade the runtime through a Council vote
              </p>
              The intention was for Jsgenesis to create a proposal for a
               
              <a
                className="Link"
                href="https://blog.joystream.org/upgrades/"
              >
                runtime upgrade
              </a>
               
              and have the Council vote on it. If the Council reached quorum, the consensus rules of the system would automatically get upgraded in flight. Unfortunately, the
               
              <a
                className="Link"
                href="https://blog.joystream.org/sparta-sacked/"
              >
                Sparta network crashed
              </a>
               
              before we reached this stage, and had to start Athens as a new chain with a new genesis block.
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>
  <div
    className="LayoutWrapper LayoutWrapper--dark"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Roles available on the current testnet
        </h2>
        
        <div
          className="ColumnsLayout"
          data-columns={2}
        >
          <a
            className="RoleCard Link"
            href="/roles#Validator"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Validator
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                At migration, 
                <span
                  className="RoleCard__counter"
                >
                  20
                </span>
                 (max) had this role
              </p>
            </div>
          </a>
          <a
            className="RoleCard Link"
            href="/roles#Council-Member"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Council Member
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                At migration, 
                <span
                  className="RoleCard__counter"
                >
                  12
                </span>
                 (max) had this role
              </p>
            </div>
          </a>
          <a
            className="RoleCard RoleCard--labeled Link"
            href="/roles#Storage-Provider"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Storage Provider
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                At migration, 
                <span
                  className="RoleCard__counter"
                >
                  10
                </span>
                 (max) had this role
              </p>
            </div>
          </a>
        </div>
      </section>
    </div>
  </div>
  <section
    className="MapInfo"
  >
    <div
      className="Map MapInfo__map"
      style={
        Object {
          "backgroundImage": "url(test-file-stub)",
        }
      }
    >
      <div
        className="Animated"
      >
        <icon-mock />
      </div>
      <p
        className="Map__info"
      >
        Map data ©
        2019
         Google
      </p>
    </div>
    <div
      className="MapInfo__content"
    >
      <div
        className="Animated"
      >
        <h1
          className="MapInfo__title"
        >
          The city of Athens
        </h1>
        <p>
          <strong>
            Athens is the capital of Greece.
          </strong>
           IIt was also at the heart of Ancient Greece, a powerful civilisation and empire. The city is still dominated by 5th-century BC landmarks, including the Acropolis, a hilltop citadel topped with ancient buildings like the colonnaded Parthenon temple.
          <br />
          <br />
          We chose the name Athens as, like our previous testnets, Mesopotamia and Sparta, the ancient Athens' historical significance in the development towards modern democracy and the rule of law.
          <br />
          <br />
          <a
            className="Link"
            href="/sparta"
          >
            <icon-mock />
             Explore previous testnet
          </a>
        </p>
      </div>
    </div>
  </section>
  <div
    className="CookiesNotice"
  >
    <div
      className="CookiesNotice__container"
    >
      <div
        className="CookiesNotice__content"
      >
        <p>
          This site uses cookies. By continuing to browse the site, you are agreeing to our use of cookies.
        </p>
        <p>
          <a
            className="CookiesNotice__link Link"
            href="/privacy-police"
          >
            Find out more ›
          </a>
        </p>
      </div>
      <button
        className="CookiesNotice__accept Button Button--secondary"
        onClick={[Function]}
        type="button"
      >
        Accept
      </button>
    </div>
  </div>
  <footer
    className="Footer"
  >
    <p
      className="Footer__header"
    >
      Get notified
    </p>
    <a
      className="Footer__button Button Button--small Button--light"
      href="http://joystream.us11.list-manage.com/subscribe/post?u=932de577aec9616d4516b4e0f&id=459ba8d1da"
      onClick={null}
    >
      Join the newsletter
    </a>
    <div
      className="Footer__socials"
    >
      <a
        className="Footer__social-wrapper"
        href="https://twitter.com/JoyStreamApp"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://github.com/Joystream"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://chat.joystream.org/home"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
    </div>
    <div
      className="Footer__links"
    >
      <a
        className="Footer__link Footer__link--highlighted Link Link--highlighted"
        href="https://www.jsgenesis.com/#openings"
        label="We are hiring!"
      >
        We are hiring!
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/"
        label="Blog"
      >
        Blog
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/manifesto/"
        label="Manifesto"
      >
        Manifesto
      </a>
      <a
        className="Footer__link Link"
        href="https://github.com/Joystream/joystream"
        label="Repository"
      >
        Repository
      </a>
    </div>
  </footer>
</div>
`;
